### PR TITLE
PWN-6855 - Change debug build type application suffix id

### PR DESCRIPTION
**Pull request by Davran331122**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-6855

### CODE REVIEW

**Comment by Davran331122** [comment]
For this task, I updated CI environment variables, for feature and debug type of build, I updated keys for feature applications.
And also I updated release keys.
Application id was added to be ensure that events will not send to release applications in analytics

**Review by 1sakov — APPROVED** [review]

**Review by 1sakov — CHANGES_REQUESTED** [review]
talked to Davran, he'll update this

File: scripts/amplitude.gradle [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        release {
+            buildConfigField("String", "amplitudeKey", apiKeysProperties.getOrThrow("amplitudeProductionKey"))
+        }
+
+        releaseLocal {
+            initWith release
+        }
+
+        releaseFirebase {
+            initWith release
```
Comment (1sakov) [inline scripts/amplitude.gradle:22]: I think that we don't need release keys for releaseFirebase/releaseLocal. 

We need production metrics for release build only

File: scripts/sentry.gradle [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        release {
+            manifestPlaceholders = [SENTRY_ENV: "release"]
+        }
+
+        releaseLocal {
+            initWith release
+        }
+
+        releaseFirebase {
+            initWith release
```
Comment (1sakov) [inline scripts/sentry.gradle:24]: Same here:
I think that we don't need release keys for releaseFirebase/releaseLocal.
We need production sentry metrics for release build only

File: scripts/amplitude.gradle [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+android {
+    buildTypes {
+        def apiKeysProperties = new PropertiesFile(rootProject.file("apikey.properties"), logger)
+        debug {
+            buildConfigField("String", "amplitudeKey", apiKeysProperties.getOrThrow("amplitudeDebugKey"))
+        }
+        feature {
+            buildConfigField("String", "amplitudeKey", apiKeysProperties.getOrThrow("amplitudeDebugKey"))
+
```
Comment (1sakov) [inline scripts/amplitude.gradle:11]: redundant line

File: scripts/amplitude.gradle [hunk 4]
```diff
@@ -0,0 +1,25 @@
+apply plugin: 'com.android.application'
+
+android {
+    buildTypes {
+        def apiKeysProperties = new PropertiesFile(rootProject.file("apikey.properties"), logger)
```
Comment (1sakov) [inline scripts/amplitude.gradle:5]: add todo about moving to a separate properties.
And did you update the environments?

**Review by 1sakov — CHANGES_REQUESTED** [review]

File: scripts/amplitude.gradle (hunk 4 above)
Comment (Davran331122) [reply scripts/amplitude.gradle:5]: For CI environments nothing was changed, I just updates CI variables

File: app/build.gradle [hunk 5]
```diff
@@ -53,12 +54,12 @@ android {
 
     buildTypes {
         debug {
+            applicationIdSuffix ".debug"
```
Comment (gslevinkov) [inline app/build.gradle:57]: do we really need a separate package for debug builds?

File: scripts/amplitude.gradle [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        debug {
```
Comment (gslevinkov) [inline scripts/amplitude.gradle:7]: is it overrides existent build or creates new one?

File: scripts/amplitude.gradle [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+        //TODO move amplitude to separate amplitude.properties file
+        def apiKeysProperties = new PropertiesFile(rootProject.file("apikey.properties"), logger)
+        debug {
+            buildConfigField("String", "amplitudeKey", apiKeysProperties.getOrThrow("amplitudeDebugKey"))
+        }
+        feature {
+            buildConfigField("String", "amplitudeKey", apiKeysProperties.getOrThrow("amplitudeDebugKey"))
+        }
+        release {
+            buildConfigField("String", "amplitudeKey", apiKeysProperties.getOrThrow("amplitudeProductionKey"))
```
Comment (1sakov) [inline scripts/amplitude.gradle:14]: `amplitudeReleaseKey`

**Comment by 1sakov** [comment]
done here - https://github.com/p2p-org/key-app-android/pull/1147

**Pull request closed without merging**